resources/events.py

```diff
@@ -42,3 +51,46 @@ def get(
         raise ErrorResponse(400, f"Could not find event: {dataset_id}/{version}")
 
     return data
+
+
+class Events(BaseModel):
+    events: List[dict]
+
+
+@router.post(
+    "",
+    dependencies=[
+        Depends(authorize("okdata:dataset:write")),
+        Depends(version_exists),
+    ],
+    responses=error_message_models(
+        status.HTTP_400_BAD_REQUEST,
+        status.HTTP_403_FORBIDDEN,
+        status.HTTP_404_NOT_FOUND,
+        status.HTTP_422_UNPROCESSABLE_ENTITY,
+        status.HTTP_500_INTERNAL_SERVER_ERROR,
+    ),
+)
+def post(
+    *,
+    dataset_id: str = Path(..., min_length=3, max_length=70, regex="^[a-z0-9-]*$"),
+    version: str = Path(..., min_length=1),
+    event_service=Depends(event_service),
+    events: Events,
+):
+    log_add(dataset_id=dataset_id, version=version)
+
+    try:
+        dataset = event_service.dataset_client.get_dataset(dataset_id)
+    except HTTPError:
+        raise ErrorResponse(
+            status.HTTP_404_NOT_FOUND, f"Dataset '{dataset_id}' not found"
+        )
+
+    try:
+        return event_service.send_events(dataset, version, events.events)
+    except PutRecordsError as e:
+        log_add(failed_records=e.num_records)
+        raise ErrorResponse(status.HTTP_500_INTERNAL_SERVER_ERROR, str(e))
```

Comment:
Ref kommentar på https://github.com/oslokommune/okdata-event-stream-api/pull/23/files#diff-bef8b82d3da0af701075aa44e9ec4fa9669989df2d39c4cbf4566fc0519244e8R28.
Istedenfor å bruke standardhåndtering for ErrorResponse så kan vi returnere en custom 500 respons her med `{message: blablabla, failed_elements: [{foo: bar}]}` slik at responsen blir mer maskinlesbar for http klienten